Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1264,6 +1268,44 @@ <h3>
     schließen kann. Weiteren Angaben über Ihre Nutzung der Corona-Warn-App erhält der Anbieter Ihres
     Betriebssystems jedoch nicht.
 </p>
+<h3>
+    o. Aktualisierungsfunktion für COVID-Zertifikate
+</h3>
+<p>
+    Es kann vorkommen, dass die App Ihnen die Aktualisierung eines gültigen digitalen
+    COVID-Zertifikats anbietet. Beschließen die EU-Mitgliedstaaten beispielsweise eine Änderung des
+    Inhalts oder der Datenstruktur der Daten im COVID-Zertifikat, kann es für die reibungslose
+    weitere Zertifikatsnutzung erforderlich sein, die jeweilige inhaltliche oder technische Änderung
+    auf ein vorhandenes digitales COVID-Zertifikat anzuwenden.
+</p>
+<p>
+    Dazu lädt die App im Hintergrund die aktuellen inhaltlichen und technischen Spezifikationen für
+    die digitalen COVID-Zertifikate vom Serversystem der App herunter. Die App überprüft dann anhand
+    der in den von Ihnen gespeicherten digitalen COVID-Zertifikaten enthaltenen Daten, ob eine
+    Aktualisierung eines gespeicherten digitalen COVID-Zertifikats über die App möglich ist. Sie
+    werden dann auf eine mögliche Aktualisierung hingewiesen. Für die Aktualisierung wird mit Ihrem
+    Einverständnis das zu aktualisierende digitale COVID-Zertifikat von der App verschlüsselt an das
+    Serversystem der App übermittelt. Wenn es sich bei dem zu aktualisierenden digitalen
```

Comment:
This part reads, as if the reissuance will only be triggered automatically by the app, the user will be notified and then the reissuing can be done once - because after receiving a reissued certificate, any additional reissuing won't be necessary anymore.
See next comment where I see a small confusion.